### ISSUE TITLE
Unable to obtain results of ResNet50 v1

### ISSUE THREAD

**Issue by netw0rkf10w** [pytorch/vision#8363]
### 🐛 Describe the bug

Hello,

I have tried using the reference classification training code to train ResNet50 on ImageNet. I would like to reproduce the results for the classical recipe (i.e., V1) with step learning rate schedule etc. What I did was executing the following:

```bash
GPUs=8
BATCH=32
MODEL=resnet50
OPT=sgd
LRSCHEDULER=steplr
LR=0.1
WD=1e-2
EPOCHS=90
torchrun --nproc_per_node=${GPUs}  train.py --model ${MODEL} --data-path ${DATA_PATH} --batch-size ${BATCH} --opt ${OPT} --lr ${LR} --lr-scheduler ${LRSCHEDULER} --epochs ${EPOCHS} --weight-decay ${WD} --norm-weight-decay 0.0  --model-ema"
```

However, the results were very bad (like 1% of validation accuracy after 6 epochs). Could you please tell me if this is expected?

Thank you very much in advance!

### Versions

PyTorch 2.2., Nvidia V100.

**Comment by netw0rkf10w** [comment]
Cc @datumbox
